TombEngine/Scripting/Internal/TEN/Objects/Lara/LaraObject.cpp

```diff
@@ -98,6 +99,81 @@ int LaraObject::GetAir() const
 	return lara->Air;
 }
 
+//Lara will undraw her weapons if they are drawn, throw away flare if she helds one in her hand.
+// @function LaraObject:UndrawWeapons
+void LaraObject::UndrawWeapons()
+{
+	auto* lara = GetLaraInfo(m_item);
+
+	if (lara->Control.HandStatus != HandStatus::Free ||
+		lara->Control.Weapon.GunType == LaraWeaponType::Flare)
+	{
+		lara->Control.HandStatus = HandStatus::WeaponUndraw;
+	}
+}
+
+//Lara will throw away the torch if she helds one in her hand.
+// @function LaraObject:ThrowAwayTorch
+void LaraObject::ThrowAwayTorch()
+{
+	auto* lara = GetLaraInfo(m_item);
+
+	if (lara->Control.Weapon.GunType == LaraWeaponType::Torch)
+	{
+		Lara.Torch.State = TorchState::Dropping;
+	}
+}
+
+// @function LaraObject:GetLaraHandStatus
+// @treturn 0=HandsFree, 1=Busy(climbing,etc), 2=WeaponDraw, 3=WeaponUndraw, 4=WeaponInHand.
+HandStatus LaraObject::GetLaraHandStatus() const
```

Comment:
Since now this function always belongs to Lara object, there is no point of calling it `GetLaraHandStatus`. Instead, it can be called `GetHandStatus`. Also, comments are formatted in a wrong way again.

Reply:
changed to GetHandStatus. Changed formatting